apache-maven/src/assembly/shared/init.ps1

```diff
@@ -0,0 +1,83 @@
+# ==== END VALIDATION ====
+
+$CLASSWORLDS_CONF = "$MAVEN_HOME\bin\m2.conf"
```

Comment:
Should this be part of `mvnlauncher.ps1` as it is never used in this file?

Reply:
I'd keep it identical to `init.cmd`.